src/frontend/screens/Game/GamePage/index.tsx

```diff
@@ -231,6 +231,7 @@ export default React.memo(function GamePage(): JSX.Element | null {
       art_square,
       art_cover,
       art_background,
+      art_logo: logo = undefined,
```

Comment:
maybe we can just use `art_logo` everywhere? it makes it easier to understand that it's the same thing that comes from the gameInfo and we already use `art_...` for the other images, makes it more consistent

Reply:
also, I think there's no need to do the `= undefined` since it will be undefined if it is undefined anyway